### PR TITLE
Adding 2024 KubeCon NA banner

### PR DESCRIPTION
**Pull request by nate-double-u**
Adding 2024 KubeCon NA banner

### Description

Adding banner between ~Oct 14~ Sept 30 and Nov 15 for KubeCon + CloudNativeCon 2024 NA

~\* Draft while confirming colors and links~

### Issue

Closes: #48085

Screenshot 2024-09-25 at 4 02 14 PM

### CODE REVIEW

**Review by pacoxu — APPROVED** [review]

**Comment by pacoxu** [comment]
/approve cancel 
sorry to approve so early.

lgtm with steering hat, and this may with for the majority met from steering.

**Comment by soltysh** [comment]
lgtm from my as well

**Comment by pohly** [comment]
LGTM (steering)

File: data/announcements/scheduled.yaml [hunk 1]
```diff
@@ -247,3 +247,18 @@ announcements:
...
+  endTime: 2024-11-15T18:00:00
+  style: >-
+    background: linear-gradient(90deg, rgba(64,66,169,1) 60%, rgba(0,81,181,1) 100%);
+    color: #fffff;
+  title: |
+    <img src ="/images/announcements/kccnc-na-2024-white.svg" style="float: right; height: 100px;" />
+    <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/">KubeCon + CloudNativeCon 2024</a>
+  message: |
+    Join us for three days of incredible opportunities to collaborate, learn and share with the cloud native community.<br/> 
+    <a href= "https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/register/?utm_source=kubernetes&utm_medium=all&utm_campaign=kubeconna24&utm_content=slim-banner">
```
Comment (nate-double-u) [inline data/announcements/scheduled.yaml:262]: Banner link update
```suggestion
    <a href= "https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/register/?utm_source=kubernetes&utm_medium=homepage&utm_campaign=KubeCon-NA-2024&utm_content=top-banner">
```

**Comment by BenTheElder** [comment]
LGTM (steering)

**Review by justaugustus — APPROVED** [review]
/lgtm
/approve (Steering)

We have 5 lgtms from Steering, so this is good to ship.

**Comment by pacoxu** [comment]
Tomorrow is Sept 30th.
/assign @tengqm

**Comment by nate-double-u** [comment]
Thanks everyone!

**Pull request merged**